nginx-docker.conf

```diff
@@ -0,0 +1,28 @@
+worker_processes  1;
+error_log  stderr warn;
+pid        /var/cache/nginx/nginx.pid;
+events {
+  worker_connections  1024;
+}
+http {
+  include       /etc/nginx/mime.types;
+  default_type  application/octet-stream;
+  log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                    '$status $body_bytes_sent "$http_referer" '
+                    '"$http_user_agent" "$http_x_forwarded_for"';
+  access_log /dev/stdout main;
+  sendfile        on;
+  keepalive_timeout  65;
+  server {
+    listen       9090;
+    server_name  localhost;
+    location / {
+      root   /app;
+      index  index.html;
```

Comment:
```suggestion
      index  index.html;
      try_files $uri $uri/ /index.html;
```

Reply:
The catch-all redirect is needed due to Hashscan's single page app URL structure.